app/routes/upsert-idcoms.js

```diff
@@ -0,0 +1,16 @@
+const { patchPackingListCheck } = require('../../app/services/dynamics-service')
+
+module.exports = {
+  method: 'GET',
+  path: '/upsert-idcoms',
+  options: {
+    handler: async (request, h) => {
+      try {
```

Comment:
We should be checking for applicationId at least is set to something useful, appreciate isParsed is conditional inside func